test/deploy/test_onnx_fft.py

```diff
@@ -299,27 +302,6 @@ def forward(self, x):
     assert torch.allclose(output.imag, output_onnx_imag, rtol, atol)
 
 
-@check_ort_version()
```

Comment:
Why did this get removed?

Reply:
This is a duplicate to the test starting on L256.

Reply:
Ah, good catch!